discretize/TensorMesh.py

```diff
@@ -159,6 +159,23 @@ def gridN(self):
         """Nodal grid."""
         return self._getTensorGrid('N')
 
+    @property
+    def h_gridded(self):
+        """Returns an M by N numpy array with the widths of all cells in order.
```

Comment:
sorry to be picky, but for docstrings that span multiple lines, would you mind using the convention of opening the docstring on one line and closing it on a separate line. 
```
"""
Returns an (nC, dim) numpy array with the widths of all cells in order
"""
```